src/pages/form/step-form/components/Step1/index.tsx

```diff
@@ -0,0 +1,125 @@
+
+const { Option } = Select;
+
+const formItemLayout = {
+  labelCol: {
+    span: 5,
+  },
+  wrapperCol: {
+    span: 19,
+  },
+};
+interface Step1Props extends FormComponentProps {
+  data?: StateType['step'];
+  dispatch?: Dispatch<any>;
+}
+
+const Step1: React.FC<Step1Props> = props => {
+  const { form, dispatch, data } = props;
+  if (!data) {
+    return null;
+  }
+  const { getFieldDecorator, validateFields } = form;
+  const onValidateForm = () => {
+    validateFields((err: any, values: StateType['step']) => {
+      if (!err && dispatch) {
+        dispatch({
+          type: 'formAndstepForm/saveStepFormData',
+          payload: values,
+        });
+        dispatch({
+          type: 'formAndstepForm/saveCurrentStep',
+          payload: 'confirm',
+        });
+      }
+    });
+  };
+  return (
+    <div className={styles['stepForm-wrapper']}>
+      <Form layout="horizontal" className={styles.stepForm} hideRequiredMark>
+        <Form.Item {...formItemLayout} label="付款账户">
+          {getFieldDecorator('payAccount', {
+            initialValue: data.payAccount,
+            rules: [{ required: true, message: '请选择付款账户' }],
+          })(
+            <Select placeholder="test@example.com">
+              <Option value="ant-design@alipay.com">ant-design@alipay.com</Option>
+            </Select>,
+          )}
+        </Form.Item>
+        <Form.Item {...formItemLayout} label="收款账户">
+          <Input.Group compact>
+            <Select defaultValue="alipay" style={{ width: 100 }}>
+              <Option value="alipay">支付宝</Option>
+              <Option value="bank">银行账户</Option>
+            </Select>
+            {getFieldDecorator('receiverAccount', {
+              initialValue: data.receiverAccount,
+              rules: [
+                { required: true, message: '请输入收款人账户' },
+                { type: 'email', message: '账户名应为邮箱格式' },
+              ],
+            })(<Input style={{ width: 'calc(100% - 100px)' }} placeholder="test@example.com" />)}
+          </Input.Group>
+        </Form.Item>
+        <Form.Item {...formItemLayout} label="收款人姓名">
+          {getFieldDecorator('receiverName', {
+            initialValue: data.receiverName,
+            rules: [{ required: true, message: '请输入收款人姓名' }],
+          })(<Input placeholder="请输入收款人姓名" />)}
+        </Form.Item>
+        <Form.Item {...formItemLayout} label="转账金额">
+          {getFieldDecorator('amount', {
+            initialValue: data.amount,
+            rules: [
+              { required: true, message: '请输入转账金额' },
+              {
+                pattern: /^(\d+)((?:\.\d+)?)$/,
+                message: '请输入合法金额数字',
+              },
+            ],
+          })(<Input prefix="￥" placeholder="请输入金额" />)}
+        </Form.Item>
+        <Form.Item
+          wrapperCol={{
+            xs: { span: 24, offset: 0 },
+            sm: {
+              span: formItemLayout.wrapperCol.span,
+              offset: formItemLayout.labelCol.span,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'offset' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)